src/temperatureConversion.js

```diff
@@ -0,0 +1,43 @@
+let preferedScale = 'celsius';
+
+const scales = {
+  celsius: {
+    conversion(x) {
+      return ((x - 32) * 5) / 9;
+    },
+    symbol: 'C',
+  },
+
+  fahrenheit: {
+    conversion(x) {
+      return (x * 9) / 5 + 32;
+    },
+    symbol: 'F',
+  },
+};
+
+function switchTemperatureScales() {
+  const available = Object.keys(scales);
```

Comment:
available does not define what is available. Should be availableScale/availableUnit...